MusicTaste/MusicTasteTests/MusicTasteTests.m

```diff
@@ -0,0 +1,36 @@
+//
+//  MusicTasteTests.m
+//  MusicTasteTests
+//
+//  Created by Aman Abraham on 7/5/22.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface MusicTasteTests : XCTestCase
+
+@end
+
+@implementation MusicTasteTests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+}
+
+- (void)testExample {
+    // This is an example of a functional test case.
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+}
```

Comment:
If you don't need this at the end, you may to clean this up by removing it later.